csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1068,6 +1068,13 @@ The list of SBOM URLs (`sbom_urls`) of value type `array` with 1 or more items c
 
 Any given SBOM URL of value type `string` with format `uri` contains a URL of one SBOM for this product.
 
+*Examples XYZ:*
```

Comment:
Only reminder for us to modify the example ordering sequence in case this gets accepted

Reply:
Tracked as #494.